package.json

```diff
@@ -2,11 +2,16 @@
   "author": "Stefan Goessner",
   "name": "jsonpath-plus",
   "version": "5.0.2",
-  "type": "commonjs",
   "main": "dist/index-node-cjs.js",
   "exports": {
-    "import": "./dist/index-node-esm.mjs",
-    "require": "./dist/index-node-cjs.js"
+    ".": {
+      "browser": "./dist/index-browser-esm.js",
+      "umd": "./dist/index-browser-umd.js",
+      "import": "./dist/index-node-esm.mjs",
+      "require": "./dist/index-node-cjs.js"
+    },
+    "./package.json": "./package.json",
+    "./": "./"
```

Comment:
Do you have a need for importing our `package.json`? (And what is `./` doing? I don't see that described as something to add.)

The other changes look good, esp. since I see no standard for "umd" (though maybe that should be set as the "default").

Reply:
> Do you have a need for importing our `package.json`? (And what is `./` doing? I don't see that described as something to add.)

I don't need them per se, it was just based on what other packages do. I'll remove them.